client/src/commands/instantiateCommand.ts

```diff
@@ -73,6 +74,10 @@ export async function instantiateSmartContract(channelTreeItem?: ChannelTreeItem
             const fabricClientConnection: IFabricConnection = FabricConnectionManager.instance().getConnection();
             await fabricClientConnection.instantiateChaincode(data.chaincode, data.version, channelName, fcn, args);
 
+            const outputAdapter: VSCodeOutputAdapter = VSCodeOutputAdapter.instance();
+            const message: string = `\nInstantiated with command: '` + fcn + `' and arguments: '` + args + `'`;
```

Comment:
you don't need the new line character also if you use backticks you can use ${variable-name} rather than closing and opening quotes